syntaxes/smithy.tmLanguage

```diff
@@ -729,7 +833,7 @@
             <string>"""</string>
             <key>endCaptures</key>
             <dict>
-                <key>0</key>
+                <key>1</key>
```

Comment:
These are supposed to start from 1? I guess this changes the capture name?

Reply:
This was over-eager on my part, I reverted that.